tools/ktid_fisher_test/kinatestid_r_fisher.xml

```diff
@@ -0,0 +1,47 @@
+<tool id="kinatestid_fisher_r" name="Kinatest-ID using Fisher's Exact Test" version="0.5.0">
+    <description>determine kinase's preferred sequence motif</description>
+    <requirements>
+       <requirement type="package">R</requirement>
+    </requirements>
+    <command><![CDATA[
+        ln -s '$substrates' substrates.csv && 
+        ln -s '$negatives' negatives.csv && 
+        ln -s '$SBF' SBF.csv &&
+        ln -s '$__tool_directory__/screener7-7.csv' screener &&
+        ln -s '$__tool_directory__/OnlyTheRequiredSubBackFreqData.RData' thedata.RData &&
+        Rscript '$__tool_directory__/7-7-fisher-galaxy_working.R'
+    ]]></command>
+    <inputs>
+        <param format="csv" name="substrates" type="data" label="Positive/Phosphorylated Substrate List"/>

```

Comment:
@Johnbbbbb can we consume and output everything in `tabular` which is tsv? Galaxy has __a lot__ of tools that can deal with tsv.

Reply:
If I'm reading this correctly, you want to combine all the outputs into a single tsv?  That's possible I guess but problem is that there is a tool after this one (ktid_scoring_function)  which will take the output of this tool.  The r code of that tool is expecting everything to be in 2 csv files, and it will look for the values to be in the correct places of the csv files.  

If you really want then I can add an *additional* output which would be "both outputs, combined into a single tsv file," but I'd really like to continue outputting the two csv files *as well* if I can, because those two csv files are then going to be used by the "ktid_scoring_function" tool and I don't want to fiddle with the R code if I don't have to

Reply:
@Johnbbbbb @bgruening is saying that the tabular format is much more common and useful within galaxy.  You would still have the same inputs and outputs, they would just be in tabular rather than csv format.